rpm/audiosystem-passthrough.spec

```diff
@@ -40,9 +40,9 @@ Binder AudioFlinger dummy service.
 %install
 rm -rf %{buildroot}
 make DESTDIR=%{buildroot} PREFIX=%{_prefix} install
-install -D -m 644 %{SOURCE1} %{buildroot}%{_libdir}/systemd/user/audiosystem-passthrough-dummy-af.service
-install -d -m 755 %{buildroot}%{_libdir}/systemd/user/user-session.target.wants
-ln -s ../audiosystem-passthrough-dummy-af.service %{buildroot}%{_libdir}/systemd/user/user-session.target.wants/audiosystem-passthrough-dummy-af.service
+install -D -m 644 %{SOURCE1} %{buildroot}%{_userunitdir}/audiosystem-passthrough-dummy-af.service
+install -d -m 755 %{buildroot}%{_userunitdir}/user-session.target.wants
+ln -s ../audiosystem-passthrough-dummy-af.service %{buildroot}%{_userunitdir}/user-session.target.wants/audiosystem-passthrough-dummy-af.service
 
```

Comment:
Should there be build requirement for systemd, macro comes from that package?

Reply:
Yes, something should pull the macros. But I'll postpone this PR until proper global solution is in place.

Reply:
We don't have any global solution for systemd macros. Also Fedora is using it by depending systemd as build requirement.